examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -0,0 +1,168 @@
+# Basic Python Continuous Integration (CI) With Cloud Source Repositories (CSR) and Cloud Build
+
+## Overview
+This repo contains example code and instructions that show you how to use CSR and 
+Cloud Build to automatically run unit tests and pylint upon code check-in. By following this tutorial you will learn
+how to build basic Python continuous integration pipelines on Google Cloud Platform (GCP).
+
+By following along with this example you will learn how to:
+1. Create a new project in CSR and clone it to your machine.
+2. Create your code and unit tests.
+3. Create a custom container called a cloud builder that Cloud Build will use to run your Python tests.
+4. Create a cloud build trigger that will tell Cloud Build when to run.
+5. Tie it all together by creating a cloudbuild.yaml file that tells Cloud Build how to execute your tests
+ when the cloud build trigger fires, using the custom cloud builder you created.
+ 
+In order to follow along with this tutorial you'll need to:
+* Create or have access to an existing [GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
```

Comment:
test

Reply:
so it seems I have the ability to start review comments but not resolve them anymore :/

Reply:
Just leave this here until I can resolve this, I'm going to try to address it. Once its fixed I'll address the changes or I'll ask someone else to finish up. Sorry for the delays @mbernico.